docs/static/releasenotes.asciidoc

```diff
@@ -13,6 +13,52 @@ This section summarizes the changes in the following releases:
 * <<logstash-6-1-1,Logstash 6.1.1>>
 * <<logstash-6-1-0,Logstash 6.1.0>>
 
+
+[[logstash-6-3-0]]
+
+[NOTE]
+--
+Persistent Queue users must upgrade. Old data will not be compatible with 6.3.0, and must be migrated or deleted. Read
+https://www.elastic.co/guide/en/logstash/current/upgrading-logstash-pqs.html[Upgrading Persistent Queue from Logstash 6.2.x and Earlier]
```

Comment:
Use shared attribute for linking:  `{logstash-ref}}/upgrading-logstash-pqs.html`
See note I added near end of page regarding location of this note. (I'm concerned that it's getting lost here, rather than being emphasized.)